staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -408,7 +417,7 @@ func RecordLongRunning(req *http.Request, requestInfo *request.RequestInfo, comp
 	if requestInfo == nil {
 		requestInfo = &request.RequestInfo{Verb: req.Method, Path: req.URL.Path}
 	}
-	var g compbasemetrics.GaugeMetric
+	var g,e compbasemetrics.GaugeMetric
```

Comment:
@jyz0309 the [test failure](https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/103799/pull-kubernetes-verify/1418216715707224064) is valid, you need to update this line:
```suggestion
	var g, e compbasemetrics.GaugeMetric
```

Reply:
Done~